#Change Node State
string RequestedNodeState
---
string NodeState
